jenkins/Jenkinsfile

```diff
@@ -15,7 +15,17 @@ pipeline {
             }
         }
         stage('Test') {
+            environment {
+                CANDLEPIN_QUAY_BOT = credentials("candlepin-quay-bot")
+            }
             parallel {
+                stage('quay-login') {
```

Comment:
I am not sure it will work if we add this stage in the parallel{} block. There might be race conditions where one of the other stages that need to do a docker pull cannot do so due to the login having not run yet. Maybe it would be better if this stage was moved out of parallel{}, between the Trust and Test stages?